src/app/services/editorial-conventions.service.ts

```diff
@@ -125,14 +125,55 @@ export class EditorialConventionsService {
         },
       },
     },
+    'sources': {
+      diplomatic: {
+        style: {
+          'font-style': 'italic',
+          'font-size': '104%',
+        },
+      },
+      interpretative: {
+        style: {
+          'font-style': 'italic',
+          'font-size': '104%',
+        },
+      },
+      critical: {
+        style: {
+          'font-style': 'italic',
+          'font-size': '104%',
+        },
+      },
+    },
+    'analogues': {
+      diplomatic: {
+        pre: '🗎',
+        style: {
+          'text-decoration': 'underline dotted from-font',
+        },
+      },
+      interpretative: {
+        pre: '🗎',
+        style: {
+          'text-decoration': 'underline dotted from-font',
+        },
+      },
+      critical: {
+        pre: '🗎',
+        style: {
+          'text-decoration': 'underline dotted from-font',
+        },
+      },
+    },
   };
 
   getLayouts(name: string, attributes: AttributesMap, defaultsKey: EditorialConventionDefaults) {
+    const excludedFromAttributeControl = ['sources', 'analogues'];
```

Comment:
What does it mean "excludedFromAttributeControl"?

Reply:
It's the bugfix we checked together; the custom editorial conventions, for some reason I don't know because I didn't make it, are not applied if the element has an attribute. So, for my new elements analogues and sources I know it's ok to apply custom style even if they have an attribute so I placed this exception